src/util/panic_hook.rs

```diff
@@ -79,6 +80,12 @@ pub fn set_exit_hook(panic_abort: bool) {
         .spawn(Backtrace::new)
         .unwrap();
 
+    // Hold the guard.
+    lazy_static! {
+        pub static ref LOG_GUARD: Mutex<Option<GlobalLoggerGuard>> = Mutex::new(None);
```

Comment:
Why make it static?

Reply:
To work around `cannot move out of captured outer variable in an `Fn` closure`.

Reply:
It's `Mutex` instead of static that fixes the error. :laughing: